### ISSUE TITLE
Split review weight into code-related and non-code related

### ISSUE THREAD

**Issue by wagnerand** [mozilla/addons-server#16274]
We'd like to split the current compound review weight into two values:
* one containing all code-related signals
* one containing all non-code-related signals.

On the review page, let's expose the the two values instead of the single combined one.

In the auto-approval queue, if we can, let's keep the combined weight and also add 2 further columns, each containing one of the new values. If that is complicated to do, we can drop the combined weight from the queue page and just expose the new values in separate columns.

**Comment by diox** [comment]
Expected. This was a request added by @wagnerand on slack.

**Comment by AlexandraMoga** [comment]
I've verified the weights split on -dev and I did not find issues. 

When an add-on generates weight after auto-approval, the Auto-approved add-ons queue will display the code-related, non-code-related and the total weights in separate columns:
![image](https://user-images.githubusercontent.com/31961530/106009409-faa47000-60c0-11eb-9247-6be11d54fb02.png)

The add-on review page will also show the split weight in the version review details:
![image](https://user-images.githubusercontent.com/31961530/106009668-41926580-60c1-11eb-8ca3-25af9f70abcf.png)

The items that fall in the non-code related category are, as far as I've noticed: abuse reports, past rejection history, admin code review, add-on reputation. I think that Bad user ratings would also figure in this category, but I did not find an add-on on -dev to match the calculation formula for bad ratings (100 bad reviews in a year to increment the weight by 2 - taken from https://github.com/mozilla/addons-server/issues/6322#issuecomment-326938760)

For the add-ons that were already in the queue, the weight has been also recalculated and split based on code and non-code related. 

Reviewer scores continue to be rewarded as before, based on the total weight value.
